src/site/_includes/learn-page.njk

```diff
@@ -25,7 +25,7 @@ pageScripts:
     <div class="wrapper auto-grid learn__collection">
       {% set coursesArr = helpers.values(courses) %}
       {% for course in coursesArr|sort(true, false, 'meta.date') %}
-      {% if course.meta and not course.meta.draft %}
+      {% if course.meta and not course.meta.draft and not course.meta.invisible %}
```

Comment:
Please double-check that condition is not required somewhere else as well.

Reply:
Had a look around and LGTM. Suspects were: tags, algolia and the courses page & it seems like the exclusions are working nicely.